src/utils.ts

```diff
@@ -58,8 +58,12 @@ export const sortPages = (arr: any, extractor: (any) => Page = n => n): any[] =>
   return arr.sort((a, b) => {
     a = extractor(a);
     b = extractor(b);
-    const aso = a.context.sidebar_order >= 0 ? a.context.sidebar_order : 10;
-    const bso = b.context.sidebar_order >= 0 ? b.context.sidebar_order : 10;
+
+    const aBase = a.context.sidebar_order ?? 10;
+    const bBase = b.context.sidebar_order ?? 10;
+    const aso = aBase >= 0 ? aBase : 10;
+    const bso = bBase >= 0 ? bBase : 10;
```

Comment:
Ah, forgot to mention this: I noticed that this was actually buggy before, because `sidebar_order` was sometimes `null` which lead to it not getting the default of 10, but null, leading to incorrect ordering etc.